### PR TITLE
Fix reuse mode env folder

### PR DESCRIPTION
**Pull request by SparkSnail**
1. Fix local reuse mode script error
~2. Fix env working folder, remove duplicated `env/{envId}`~
3. Fix reuse mode temp folder, remove duplicated `{experimentId}`

### CODE REVIEW

File: ts/nni_manager/training_service/reusable/trialDispatcher.ts [hunk 1]
```diff
@@ -651,13 +651,13 @@ class TrialDispatcher implements TrainingService {
...
         this.log.info(`Assign environment service ${environmentService.getName} to environment ${envId}`);
-        environment.command = `sh ../install_nni.sh && python3 -m nni.tools.trial_tool.trial_runner`;
+        environment.command = `sh install_nni.sh && python3 -m nni.tools.trial_tool.trial_runner`;
 
         if (this.isDeveloping) {
             environment.command = "[ -d \"nni_trial_tool\" ] && echo \"nni_trial_tool exists already\" || (mkdir ./nni_trial_tool && tar -xof ../nni_trial_tool.tar.gz -C ./nni_trial_tool) && pip3 install websockets && " + environment.command;
         }
 
-        environment.command = `mkdir -p envs/${envId} && cd envs/${envId} && ${environment.command}`;
+        environment.command = `${environment.command}`;
```
Comment (J-shang) [inline ts/nni_manager/training_service/reusable/trialDispatcher.ts:660]: seems this command is running outside `envs`, do we need ```cd envs && ${environment.command}``` ?

**Comment by liuzhe-lz** [comment]
Btw, do we have AML test case? I can't find one.

File: ts/nni_manager/training_service/reusable/trialDispatcher.ts (hunk 1 above)
Comment (SparkSnail) [reply ts/nni_manager/training_service/reusable/trialDispatcher.ts]: yes, I reverted the change here.

**Comment by SparkSnail** [comment]
> Btw, do we have AML test case? I can't find one.

No, we don't have AML tests yet.

File: ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts [hunk 2]
```diff
@@ -264,7 +264,7 @@ ${environment.command} --job_pid_file ${environment.runnerWorkingFolder}/pid \
         }
         const executor = await this.getExecutor(environment.id);
         const environmentLocalTempFolder: string =  
-            path.join(this.experimentRootDir, this.experimentId, "environment-temp")
+            path.join(this.experimentRootDir, "environment-temp")
```
Comment (J-shang) [inline ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts:267]: seems we don't need `cd ${environment.runnerWorkingFolder}` in L253, due to we have `cd envs/${envId}` in `trialDispatcher.ts`.

**Review by liuzhe-lz — APPROVED** [review]

File: ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts (hunk 2 above)
Comment (SparkSnail) [reply ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts:267]: Need to `cd ${environment.runnerWorkingFolder}` to set script root dir, other wise script will executed based on wrongly directory. This pr only fix local-temp folder, reverted env related change.

**Review by J-shang — APPROVED** [review]

**Pull request merged**